frontend/components/utils/FormatCurrency.vue

```diff
@@ -34,6 +36,19 @@ export default Vue.extend({
         },
     },
     computed: {
+        formattedNumber(): number {
```

Comment:
If this is called `formattedNumber`, it should return string, not number, otherwise you can end up displaying some very long number

Reply:
Whats a good name I could use otherwise?

Reply:
Sorry that it's not clear, I implied not to rename the function, but to always return string